Kitodo/pom.xml

```diff
@@ -52,38 +52,6 @@
             <artifactId>kitodo-ugh</artifactId>
             <version>3.0-SNAPSHOT</version>
         </dependency>
-        <dependency>
-            <groupId>avalon-framework</groupId>
-            <artifactId>avalon-framework</artifactId>
-            <version>4.1.5</version>
-        </dependency>
-        <!-- https://mvnrepository.com/artifact/cglib/cglib -->
-        <dependency>
-            <groupId>cglib</groupId>
-            <artifactId>cglib</artifactId>
-            <version>2.2</version>
-        </dependency>
-        <dependency>
-            <groupId>com.keypoint</groupId>
-            <artifactId>png-encoder</artifactId>
-            <version>1.5</version>
-        </dependency>
-        <dependency>
-            <groupId>jfree</groupId>
-            <artifactId>jcommon</artifactId>
-            <version>1.0.0</version>
-            <exclusions>
-                <exclusion>
-                    <groupId>junit</groupId>
-                    <artifactId>junit</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>
-        <dependency>
```

Comment:
This library is bring back to the project by UGH.

Reply:
Thank you. I will re-add this dependency as it is needed.